mlrun/artifacts/model.py

```diff
@@ -231,13 +231,13 @@ def get_model(model_dir, suffix=""):
     if obj.kind == "file":
         return model_file, model_spec, extra_dataitems
 
-    tmp = mktemp(suffix)
-    obj.download(tmp)
-    return tmp, model_spec, extra_dataitems
+    temp_path = tempfile.NamedTemporaryFile(suffix=suffix, delete=False).name
+    obj.download(temp_path)
+    return temp_path, model_spec, extra_dataitems
```

Comment:
`get_model` is a function the client may use, you can't just add a new value in the returned tuple, that is non backwards compatible

Reply:
I did not add a new value, I renamed an existing one `tmp` -> `temp_path`